### ISSUE TITLE
testting summary stats

### ISSUE THREAD

**Issue by richfitz** [mwpennell/arbutus#6]
_Issue by **[mwpennell](https://github.com/mwpennell)** from Friday Sep 27, 2013 at 02:19 GMT_
_Originally opened as https://github.com/richfitz/modeladequacy/issues/15_

---

I am not sure of the best way to evaluate the summary statistic fxn. Currently just using expected values on the geospiza dataset. This will only help if I change things as the calculations are based on current code. However, difficult to test independently because involves some complicated fxns like lm and ks.test. not sure best way to do this.

**Comment by diogoprov** [comment]
Hi @richfitz and @mwpennell  just a dumb question: What do the abbreviations of the summary statistics stand for, as returned by function arbutus?

**Comment by wcornwell** [comment]
Not a dumb question, this is the same problem as #17 -- be nice to have this info somewhere easy to find

**Comment by diogoprov** [comment]
Yes, the way the help file of the function is written now doesn't facilitate this. Maybe having a "Value" section in the help explaining this would be nice indeed. Thanks!
